modules/soc_http/main.sh

```diff
@@ -31,11 +45,41 @@ socDebugLog(){
 	fi
 }
 
-socDebugLog "Requesting SoC"
-soc=$(curl --connect-timeout 15 -s $ip | cut -f1 -d".")
+getAndWriteSoc(){
+	re='^-?[0-9]+$'
+	openwbDebugLog ${DMOD} 0 "Lp$CHARGEPOINT: Requesting SoC"
+	echo 0 > $soctimerfile
+	soc=$(curl --connect-timeout 15 -s $ip | cut -f1 -d".")
```

Comment:
Die Variable $soc wird nie eine Fehlermeldung enthalten, da das Ergebnis des curl Aufrufs durch jq geleitet wird. Die folgenden Meldungen in Zeilen 60 und 64 bringen somit keinen Mehrwert. Am Besten das Ergebnis von curl inklusive Fehlermeldungen zwischenspeichern und im zweiten Schritt mit jq parsen.
`result=$(curl .... 2>&1)
soc=(echo "$result" | cut ...)`
Bei einem Fehler enthält dann $result alle Infos.

Reply:
Hier bräuchte ich Schützenhilfe - so fit bin ich hier nicht. Könntest Du die Änderung ausformulieren, so dass ich diese übernehme und den PR update?

Reply:
Nimm aus den nachfolgenden Fehlermeldungen vorerst einfach die Variable $soc raus. Ich teste dann nach dem Merge etwas damit und ergänze die Fehlermeldung.